include/mc_rbdyn/lipm_stabilizer/StabilizerConfiguration.h

```diff
@@ -577,6 +580,7 @@ struct MC_RBDYN_DLLAPI StabilizerConfiguration
 
       if(tasks.has("contact"))
       {
+        tasks("contact")("constrainCoP", constrainCoP);
```

Comment:
Should be documented here: https://github.com/jrl-umi3218/mc_rtc/blob/master/doc/_data/schemas/common/contact.json

Also this might be a little awkward to put it there, maybe it should be at the root of the configuration?